schema/CompoundSchema.mcf

```diff
@@ -26,6 +26,12 @@ subClassOf: dcs:ChemicalCompound
 description: "Compound that is inadvisable to be used to treat a given disease."
 url: "https://www.ebi.ac.uk/chembl/"
 
+Node: dcid:ChemicalCompoundGeneAssociation
+name: "ChemicalCompoundGeneAssociation"
+typeOf: schema:Class
+subClassOf: schema:Intangible
```

Comment:
I'm conflicted on this. dcs:GeneticAssoication could work. However, it's a subClass of BiologicalElement and thus far we've been treating BiologicalElement and ChemicalCompound as two separate trees. Here they are combining the two. I'm more inclined to not put it under either tree. What do you think @tjann

Reply:
Ah I see. I saw Gene and thought BiologicalElement would be good.

Maybe it's still good to create an umbrella class. Say, ChemicalElement, BioChemicalElement.. or if not Element, Concept/Thing?

Reply:
I think we should just make it either a Thing or make a BiomedicalElement (subclass of Thing) that BiologicalElement and ChemicalElement. I'm liking the second option, because I could also see making a MedicalElement class in the future as well, which would fall under the BiomedicalElement umbrella.

Reply:
Let's make a new class BiomedicalElement as a subclass of Thing and then add this and BiologicalElement and ChemicalElement as a subclass of BiomedicalElement.